packages/browser/src/integrations/launchdarkly.ts

```diff
@@ -0,0 +1,69 @@
+import type { Client, Event, EventHint, IntegrationFn } from '@sentry/types';
+import type { LDContext, LDEvaluationDetail, LDInspectionFlagUsedHandler } from 'launchdarkly-js-client-sdk';
```

Comment:
I think the SDK team wants you to inline these types instead of having this as a dev dep

Reply:
Ok, what are the good practices for that? Do I need to exactly re-define the whole type, or could I just put like `object` for LDContext and LDEvaluationDetail

Reply:
It doesn't need to be exact, but it should be more specific than `object` (not very helpful I know).

Reply:
Is there a reason we prefer not to have the dev dep? It doesn't affect bundle size right

Reply:
1. it causes security vuln scanner to scream false positives (most important)
2. some users do full installs (which also install dev deps), which means it effects download times and gives wrong impression for bundle size